index.html

```diff
@@ -4,50 +4,118 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
     <link rel="stylesheet" href="style.css">
     <title>Components</title>
 </head>
 <body>
-    <h1>Avatar Components</h1>
```

Comment:
It could be better if you wrap your html code inside a div. 
Otherwise everything is good to commit.

Reply:
yes for documentation I will do that and for individual items I felt it is not required, Thank you for the review